src/st/class-wpml-pb-loader.php

```diff
@@ -69,4 +63,13 @@ public function __construct(
 		}
 
 	}
+
+	private static function load_hooks() {
+		$hooks = [
+			WPML_PB_Handle_Post_Body::class,
+			WPML\PB\Compatibility\Toolset\Layouts\HooksFactory::class,
```

Comment:
I suggest use statement

Reply:
I prefer to keep this @gregWierzba. This is the main loader class and if we simplify the class name, we will end up with aliases for several `HooksFactory` classes. I think it's better to keep the full name here.